processing/src/test/java/org/apache/druid/query/groupby/NestedQueryPushDownTest.java

```diff
@@ -584,6 +590,70 @@ public void testDimensionFilterOnOuterQuery()
     Assert.assertEquals(expectedRow0, results.get(0));
   }
 
+  @Test
+  public void testVirtualColumnFilterOnInnerQuery()
```

Comment:
This class is supposed to test groupBy queries with `GroupByQueryConfig.CTX_KEY_FORCE_PUSH_DOWN_NESTED_QUERY` enabled. Could you please move this test to somewhere else such as `GroupByQueryRunnerTest`?

Reply:
done